src/index.ts

```diff
@@ -8,16 +8,20 @@ export interface BrowserTransportOptions {
     classPrefix?: string
     /** Whether to inject CSS styles in the page header, defaults to true. */
     injectStyles?: boolean
+    /** Whether to display request success and error messages */
```

Comment:
We should say what the default value for this is in the docstring

Reply:
Added the defaults to the comment!